psi4/src/psi4/libqt/qt.h

```diff
@@ -109,7 +109,7 @@ void C_DROT(size_t ntot, double* x, int incx, double* y, int incy, double costhe
 void C_DSWAP(size_t length, double* x, int incx, double* y, int inc_y);
 void C_DSCAL(size_t len, double alpha, double* vec, int inc);
 void C_DCOPY(size_t length, double* x, int inc_x, double* y, int inc_y);
-void C_DAXPY(size_t length, double a, double* x, int inc_x, double* y, int inc_y);
+void C_DAXPY(size_t length, double a, const double* x, int inc_x, double* y, int inc_y);
```

Comment:
We normally omit the `const` in the BLAS/LAPACK wrappers (even though input-only args are not touched), because the underlying Fortran functions can't provide const safety.  It might be a good idea to omit it here, for consistency.

Reply:
I can do that, but the price is losing a `const` qualifier on `arrayPtr`.

Reply:
Entirely up to you - just wanted to point out the slight inconsistency.  If it introduces the need for annoying casts everywhere in the calling code, it's better to stick with the `const` in the signature.